vuln/npm/390.json

```diff
@@ -16,5 +16,6 @@
   "recommendation": "update simple-server to 1.1.0 or higher",
   "references": "- https://www.hackerone.com/reports/309641",
   "cvss_vector": "CVSS:3.0/AV:U/AC:U/PR:U/UI:U/S:U/C:U/I:U/A:U",
```

Comment:
Isn't this basically an «Unknown» vector? If yes, where did you get the score from?
Perhaps also update the vector, if you made it?